src/test/java/com/wikia/webdriver/testcases/auth/ForcedLoginTests.java

```diff
@@ -46,11 +46,18 @@ public void anonCanLogInViaUserLoginPage() {
     assertTrue(base.isStringInURL(URLsContent.SPECIAL_UPLOAD));
   }
 
+  @Test(invocationCount = 5)
   public void anonCanLogInOnSpecialWatchListPage() {
     WikiBasePageObject base = new WikiBasePageObject();
     base.openSpecialWatchListPage(wikiURL);
     base.clickLoginOnSpecialPage();
     new AttachedSignInPage().login(user);
+    //TODO: To delete
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)